components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -620,7 +617,7 @@ func (r *Reconciler) CreateOrUpdatePublisherProxy(ctx context.Context, backend e
 
 	// Update publisher proxy deployment
 	if err := r.Update(ctx, desiredPublisher); err != nil {
-		return nil, errors.Wrapf(err, "update publisher proxy deployment failed")
+		return nil, errors.Wrapf(err, "update Event Publisher deployment failed")
```

Comment:
`failed to...` ???

Reply:
https://github.com/kyma-project/kyma/pull/14725#discussion_r912035043